build/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:alpine as builder
+
+RUN apk add gcc libc-dev
+# Create initial dir
+RUN mkdir archon
+COPY ./ ./archon
+WORKDIR archon
+# Compile the code
+RUN mkdir .bin
+ENV GOBIN=/go/archon/.bin
+RUN go install ./cmd/*
+# Create a directory for the server files
+RUN mkdir archon_server
+WORKDIR archon_server
+RUN cp ../.bin/* .
+# Copy the supporting files
+RUN cp -r ../setup/* .
+# Overwrite config
+COPY /build/docker_config.yaml ./config.yaml
+# Generate certificate
+RUN ./generate_cert -ip 0.0.0.0/32
```

Comment:
I don't recall whether or not this is a valid SAN, I think I had trouble with this in the past. Does CIDR work?

Reply:
Hm. I didn't seem to have any issues with this. I've actually used the value from source code comment in `generate_cert/main.go` :

> // If you want to make your life a little easier (albeit technically less secure),
> // use 0.0.0.0/32 as the address.

I can change this to something else but I'm not quite sure what would be better.

Reply:
Nah if it's working for you then great; I've just been using smaller scoped certs for development work. Would be great to use a wider range instead